frontend/components/dashboard/chart/SummaryChart.vue

```diff
@@ -221,14 +323,140 @@ const option = computed(() => {
     }
   }
 })
+
+// get the current dataZoom settings in the chart
+const getDataZoomValues = () => {
+  const chartOptions = chart.value?.getOption()
+  const start: number = get(chartOptions, 'dataZoom[0].start', 80) as number
+  const end: number = get(chartOptions, 'dataZoom[0].end', 100) as number
+  return {
+    start,
+    end
+  }
+}
+
+// get the from to values for the selected zoom settings
+const getZoomTimestamps = () => {
+  const max = categories.value.length - 1
+  if (max <= 0) {
+    return
+  }
+  const zoomValues = getDataZoomValues()
+  const toIndex = Math.ceil(max / 100 * zoomValues.end)
+  const fromIndex = Math.floor(max / 100 * zoomValues.start)
+  return {
+    ...zoomValues,
+    toIndex,
+    toTs: categories.value[toIndex],
+    fromIndex,
+    fromTs: categories.value[fromIndex]
+  }
+}
+
+// validate and adjust zoom settings
+const validateDataZoom = (instant?: boolean) => {
```

Comment:
When I go too far in the past, the chart becomes empty and I think that that it would confuse users:
![image](https://github.com/user-attachments/assets/f236fd7d-6889-494c-8d49-aa2155622d32)
Maybe we want to show an error tooltip or a dialog inviting to subscribe?